mne/evoked.py

```diff
@@ -526,7 +526,7 @@ def get_peak(self, ch_type=None, tmin=None, tmax=None,
             Defaults to 'abs'.
         time_as_index : bool
             Whether to return the time index instead of the latency in seconds.
-        merge_grads : bool
+        merge_channels : bool
             If True, compute peak from merged gradiometer data.
```

Comment:
same here. And docstrings should then be updated. To be honest I would revert this one unless you need it already for NIRS